# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrdao/x86_64-support.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

x86-64 support - imported from ROCK Linux.

  - Rene Rebe <rene@exactcode.de>


--- ./scsilib/RULES/x86_64-linux-cc.rul.old	1970-01-01 01:00:00.000000000 +0100
+++ ./scsilib/RULES/x86_64-linux-cc.rul	2004-12-28 14:57:29.000000000 +0100
@@ -0,0 +1,70 @@
+#ident "@(#)i586-linux-cc.rul	1.5 02/10/15 "
+###########################################################################
+# Written 1996 by J. Schilling
+###########################################################################
+#
+# Platform dependent MACROS for Linux
+#
+###########################################################################
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; see the file COPYING.  If not, write to
+# the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+###########################################################################
+include $(SRCROOT)/$(RULESDIR)/rules.prg
+###########################################################################
+
+CPPFLAGS=	$(CPPOPTS) $(CPPOPTX)
+CFLAGS=		$(COPTS) $(CWARNOPTS) $(COPTOPT) $(GCCOPTOPT) $(COPTX)
+
+CPPOPTS=	-I. -I$(ARCHDIR) -I$(OINCSDIR) $(INCDIRS:%=-I%) $(OSDEFS)
+COPTS=
+CWOPTS=		-Wall -Wtraditional \
+		-Wshadow -Wmissing-prototypes -Wstrict-prototypes
+COPTOPT=	-O
+KDEFINES=	-DKERNEL -D_KERNEL
+COPTDYN=	-fPIC
+COPTGPROF=	-pg
+
+LIB_PREFIX=	lib
+LIB_SUFFIX=	.a
+SHL_SUFFIX=	.so.1.0
+
+#LIB_SOCKET=	-lsocket -lnsl -ldl
+LIB_SOCKET=	
+LIB_MATH=	-lm
+#LIB_KVM=	-lkvm
+LIB_KVM=	
+
+LIBS_PATH=	-L$(OLIBSDIR)
+
+LDFLAGS=	$(LDOPTS) $(LDOPTX)
+LDLIBS=		$(LIBS) $(LIBX)
+
+#LDOPTS=		$(LIBS_PATH) $(LDPATH) $(RUNPATH)
+LDOPTS=		$(LIBS_PATH) $(LDPATH)
+LDOPTDYN=	-shared -Wl,-soname,$(TARGET)
+LNDYNLIB=	@$(SYMLINK) $(TARGET) $(PTARGET_BASE).so
+
+FLOAT_OPTIONS=	
+
+CC=		@echo "	==> COMPILING \"$@\""; gcc
+LDCC=		@echo "	==> LINKING \"$@\""; gcc
+DYNLD=		@echo "	==> LINKING dynamic library \"$@\""; gcc
+RANLIB=		@echo "	==> RANDOMIZING ARCHIVE \"$@\""; true
+ARFLAGS=	cr
+LORDER=		echo
+TSORT=		cat
+
+RMDEP=		:
+MKDEP=		@echo "	==> MAKING DEPENDENCIES \"$@\""; $(RMDEP); gcc -M
+MKDEP_OUT=
